vision/samples/v1/samplegen_resource_path.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_resource_path")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-vision
+
+# sample-metadata
+#   title: Create product set (demonstrate resource paths)
+#   description: Create product set (demonstrate resource paths)
+#   usage: python3 samples/v1/samplegen_resource_path.py [--project "[PROJECT ID]"]
+
+# [START samplegen_resource_path]
+from google.cloud import vision_v1
+
+
+def sample_create_product_set(project):
+    """
+    Create product set (demonstrate resource paths)
+
+    Args:
+      project The Google Cloud Project for creating this product set
+    """
+
+    client = vision_v1.ProductSearchClient()
+
+    # project = '[PROJECT ID]'
```

Comment:
Instead of commenting these out, can we use default values the user can replace?

This way they get compile checked. 
```python
def sample_create_product_set(project="YOUR_DISPLAY_NAME"):
```

Reply:
I'm ok with this because it's how we do it in node and I'm used to it, but I'd love to hear from more folks as a tie breaker (do we leave it uncommented in java?)

Reply:
For both Go and Java we switch from commenting them out to using an overriding function to provide defaults. The issue with commented out variables is that there isn't a way to check them vs the compiler, so they often could get changed or updated without anyone noticing.

Reply:
Same exact thing in both Ruby and C# as well 😄

We're switching from comments to using method parameter default values.

Love it +💯